### ISSUE TITLE
Cannot check version for RPM >= 4.15 

### ISSUE THREAD

**Issue by dtanzer** [electron-userland/electron-installer-redhat#163]
**What version of `electron-installer-redhat` are you using?**

3.0.0

**What version of `node` and `npm` are you using?**

node v14.3.0  
npm 6.14.5

**What operating system are you using?**

Linux - Fedora 32 (Workstation Edition)

**Can you compile the [example app](https://github.com/unindented/electron-installer-redhat/tree/master/example) successfully?**

Haven't tried

**If not, paste here the output of the following commands:**

```
$ git clone https://github.com/unindented/electron-installer-redhat.git
$ cd electron-installer-redhat/example
$ DEBUG='electron-installer-redhat' npm run build
```

n/a

**What did you do? Please include the configuration you are using for `electron-installer-redhat`.**

        {
          "name": "@electron-forge/maker-rpm",
          "config": {
            "options": {
              "icon": "./assets/images/[some-image].png"
            }
          }
        }

**What did you expect to happen?**

a RPM to be built

**What actually happened?**

The installer crashes because the RPM version output cannot be parsed; `RPM-Version 4.15.1` contains only one space, the installer expects 2.

**Comment by fcastilloec** [comment]
This is being addressed in #162

**Comment by dtanzer** [comment]
@fcastilloec I know, see my comment on the PR: https://github.com/electron-userland/electron-installer-redhat/pull/162#issuecomment-635755650
